lambda_src/drivers/destination_s3.py

```diff
@@ -82,15 +82,17 @@ def write_to_s3(bucket: Text, filename: Text, content: AnyStr) -> Dict[Text, Any
     )
 
 
-def initialize(destination, batch_id: Text):
```

Comment:
Removed the `batch_id` arg, as it becomes vestigial since we no longer use it while initializing the folder

Reply:
this is a generic API so just because it's vestigial here doesn't mean it won't be useful in another destination driver